fenix/app/src/main/java/org/mozilla/fenix/library/history/HistoryFragmentStore.kt

```diff
@@ -110,8 +111,11 @@ fun HistoryMetadata.toHistoryMetadata(position: Int): History.Metadata {
 /**
  * The [Store] for holding the [HistoryFragmentState] and applying [HistoryFragmentAction]s.
  */
-class HistoryFragmentStore(initialState: HistoryFragmentState) :
-    Store<HistoryFragmentState, HistoryFragmentAction>(initialState, ::historyStateReducer)
+class HistoryFragmentStore(
```

Comment:
Should this eventually be renamed to `HistoryStore`?

Reply:
Potentially! There are other areas within the History feature - there is an `HistorySearchFragmentStore` and the `HistoryMetadataFragment` will need a Store eventually as well, for example. I would probably hold off on renaming until those could be all consolidated, if ever.